src/options.ts

```diff
@@ -7,6 +12,9 @@ document.addEventListener('DOMContentLoaded', e => {
         .forEach((x: HTMLInputElement) => {
             activeFeatures[x.id] = x.checked;
         });
+    chrome.storage.local.get('tokens', object => {
+        canvasTextbox.value = object.tokens.canvas;
+    });
```

Comment:
```suggestion
    chrome.storage.local.get('tokens', ({tokens}) => {
        canvasTextbox.value = tokens.canvas;
    });
```
Same simplification as above.